<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="1" failed="0" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-02-23T17:28:18 PET" name="Data Driven Basics" finished-at="2023-02-23T17:32:31 PET" duration-ms="252770">
    <groups>
    </groups>
    <test started-at="2023-02-23T17:28:18 PET" name="App Manager Login Test" finished-at="2023-02-23T17:32:31 PET" duration-ms="252770">
      <class name="org.meto.appium.tests.TFLogin">
        <test-method is-config="true" signature="setup()[pri:0, instance:org.meto.appium.tests.TFLogin@6c0d9d86]" started-at="2023-02-23T17:28:18 PET" name="setup" finished-at="2023-02-23T17:28:38 PET" duration-ms="19742" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="FillForm()[pri:0, instance:org.meto.appium.tests.TFLogin@6c0d9d86]" started-at="2023-02-23T17:28:38 PET" name="FillForm" finished-at="2023-02-23T17:32:31 PET" duration-ms="232400" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- FillForm -->
        <test-method is-config="true" signature="preSetup()[pri:0, instance:org.meto.appium.tests.TFLogin@6c0d9d86]" started-at="2023-02-23T17:28:38 PET" name="preSetup" finished-at="2023-02-23T17:28:38 PET" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preSetup -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:org.meto.appium.tests.TFLogin@6c0d9d86]" started-at="2023-02-23T17:32:31 PET" name="teardown" finished-at="2023-02-23T17:32:31 PET" duration-ms="589" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "io.appium.java_client.service.local.AppiumDriverLocalService.stop()" because "this.service" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "io.appium.java_client.service.local.AppiumDriverLocalService.stop()" because "this.service" is null
at org.meto.appium.base.TestBase.teardown(TestBase.java:57)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:361)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:296)
at org.testng.internal.invokers.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:222)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:131)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- org.meto.appium.tests.TFLogin -->
    </test> <!-- App Manager Login Test -->
  </suite> <!-- Data Driven Basics -->
</testng-results>
